src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        Ball chosenOne = new Ball();
+        chosenOne.setColor(getRandomColor());
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);

```

Comment:
Creating a new instance of `Random` every time the method is called is inefficient. Consider making it a private class-level field to reuse the same instance across multiple calls.